app/jobs/reindex_job.rb

```diff
@@ -3,33 +3,25 @@
 # Reindexes an object
 class ReindexJob
   include Sneakers::Worker
-  include Dry::Monads[:result]
```

Comment:
Cleaner interface to Indexer means that consumers of the class don't need to muck with monads.